kubernetes/samples/scripts/create-fmw-infrastructure-domain/create-domain-inputs.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+# The version of this inputs file.  Do not modify.
+version: create-weblogic-sample-domain-inputs-v1
+
+# Port number for admin server
+adminPort: 7001
+
+# Name of the Admin Server
+adminServerName: admin-server
+
+# Unique ID identifying a domain.
+# This ID must not contain an underscope ("_"), and must be lowercase and unique across all domains in a Kubernetes cluster.
+domainUID: domain1
+
+# Home of the WebLogic domain
+# If not specified, the value is derived from the domainUID as /shared/domains/<domainUID>
+domainHome: /shared/domains/domain1
+
+# Determines which WebLogic Servers the operator will start up
+# Legal values are "NEVER", "IF_NEEDED", or "ADMIN_ONLY"
+serverStartPolicy: IF_NEEDED
+
+# Cluster name
+clusterName: cluster-1
+
+# Number of managed servers to generate for the domain
+configuredManagedServerCount: 5
+
+# Number of managed servers to initially start for the domain
+initialManagedServerReplicas: 2
+
+# Base string used to generate managed server names
+managedServerNameBase: managed-server
+
+# Port number for each managed server
+managedServerPort: 8001
+
+# WebLogic Server Docker image.
+# The operator requires WebLogic Server 12.2.1.3.0 with patch 29135930 applied.
+# The existing WebLogic Docker image, `store/oracle/weblogic:12.2.1.3`, was updated on January 17, 2019,
+# and has all the necessary patches applied; a `docker pull` is required if you already have this image.
+# Refer to [WebLogic Docker images](../../../../../site/weblogic-docker-images.md) for details on how
+# to obtain or create the image.
+image: oracle/fmw-infrastructure:12.2.1.3
```

Comment:
It seems like the image name and the comments above are not in 100% alignment? As in the comments would reference the FMW related paths and GitHub site docs?

Reply:
fixed